### PR TITLE
[AMORO-2213] Expose table operations REST API

### LINKED ISSUE #2213: [Improvement]: Add table operation REST API
### Search before asking

- [X] I have searched in the [issues](https://github.com/NetEase/amoro/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

The table operations are not exposed through REST API. This PR fixes the problem.

### How should we improve?

Add the corresponding REST handler to DashboardServer.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Subtasks

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by link3280**
## Why are the changes needed?
Add the corresponding REST handler to DashboardServer.

Close #2213.

## Brief change log
As titled.

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? no
- If yes, how is the feature documented? not documented

### CODE REVIEW

File: ams/server/src/main/java/com/netease/arctic/server/dashboard/DashboardServer.java [hunk 1]
```diff
@@ -141,229 +141,232 @@ public EndpointGroup endpoints() {
...
+            tableController::getTransactionDetail);
+          get(
+            "/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/partitions",
+            tableController::getTablePartitions);
+          get(
+            "/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/partitions/{partition}/files",
+            tableController::getPartitionFileListInfo);
+          get(
+            "/tables/catalogs/{catalog}/dbs/{db}/tables/{table}/operations",
+            tableController::getTableOperations);
```
Comment (link3280) [inline ams/server/src/main/java/com/netease/arctic/server/dashboard/DashboardServer.java:307]: This is the main change. Other changed lines are due to code reformat.

**Comment by zhoujinsong** [comment]
The Amoro Project has switched to using Spotless for automatic code formatting and checking. You can execute the command `mvn spotless:apply` to automatically format the codes.

**Comment by link3280** [comment]
@zhoujinsong Thanks for the pointer! The PR is updated. Should be a quick one.

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Review by wangtaohz — APPROVED** [review]
LGTM

**Pull request merged**